go/chat/utils/utils.go

```diff
@@ -886,25 +886,10 @@ func PresentMessageUnboxed(ctx context.Context, rawMsg chat1.MessageUnboxed, uid
 	if err != nil {
 		return miscErr(err)
 	}
-
 	switch state {
 	case chat1.MessageUnboxedState_VALID:
-		if !rawMsg.IsValidFull() {
-			return miscErr(fmt.Errorf("unexpected deleted %v message", strings.ToLower(rawMsg.GetMessageType().String())))
```

Comment:
So you _do_ want deleted messages to be able to get up to the frontend? With blank `MessageBody` fields?

Reply:
I guess not, I'll add that back in.